tests/strategy/test_strategy_helpers.py

```diff
@@ -177,26 +177,30 @@ def test_stoploss_from_open(side, profitrange):
     ("long", 0.1, 0.2, 1, 0.08333333),
     ("long", 0.1, 0.5, 1, 0.266666666),
     ("long", 0.1, 5, 1, 0.816666666),  # 500% profit, set stoploss to 10% above open price
+    ("long", 0, 5, 10,  3.3333333),  # 500% profit, set stoploss break even
+    ("long", 0.1, 5, 10,  3.26666666),  # 500% profit, set stoploss to 10% above open price
+    ("long", -0.1, 5, 10,  3.3999999),  # 500% profit, set stoploss to 10% belowopen price
 
     ("short", 0, 0.1, 1, 0.1111111),
     ("short", -0.1, 0.1, 1, 0.2222222),
     ("short", 0.1, 0.2, 1, 0.125),
     ("short", 0.1, 1, 1, 1),
+    ("short", -0.01, 5, 10, 10.01999999),  # 500% profit at 10x
```

Comment:
assuming the below sample of an open-rate at 100$ - what's the expectation with `stoploss_from_open(-0.01, curr_profit)` at 10x?

The current implementation will yield 100.1 - which results in 1% "own capital at risk" below the open (1% / 10x).
as this is "stoploss_from_open" - this could also be interpreted as "price distance" - but i feel that would be strange behavior, as all other stops are  leverage adjusted.

Reply:
I see the code annotation for  `open_relative_stop` argument is `Desired stop loss percentage relative to open price`, so I feel the same that  `stops with leverage adjusted`  is more suitable than `price distance`. And It's can also be consistent with other stops.